tools/disasm/variables.txt

```diff
@@ -11572,7 +11572,7 @@
     0x80A4527C:("D_80A4527C","UNK_TYPE1","",0x1),
     0x80A45290:("D_80A45290","UNK_TYPE1","",0x1),
     0x80A452A4:("D_80A452A4","UNK_TYPE1","",0x1),
-    0x80A452B0:("D_80A452B0","UNK_TYPE4","",0x4),
+    0x80A452B0:("colHeader","UNK_TYPE4","",0x4),
```

Comment:
```suggestion
    0x80A452B0:("D_80A452B0","UNK_TYPE4","",0x4),
```
Probably best to just leave this as is, or at least give it the same name you gave it in the actor.